index.js

```diff
@@ -521,6 +544,11 @@
     return typeof other === typeof this && other.valueOf() === this.valueOf();
   }
 
+  //  Boolean$prototype$lte :: Boolean ~> Boolean -> Boolean
+  function Boolean$prototype$lte(other) {
+    return typeof other === typeof this && this.valueOf() <= other.valueOf();
+  }
+
```

Comment:
After giving this matter some thought, I suggest:

  - we consider `false` and `true` the two members of `BooleanPrimitive`;
  - we consider `new Boolean(false)` and `new Boolean(true)` the two members of `BooleanObject`; and
  - we have `Boolean$prototype$lte` define `fantasy-land/lte` for both `BooleanPrimitive` and `BooleanObject`.

Comparing a `BooleanPrimitive` to a `BooleanObject` (via `equals` or `lte`) will always result in `false`.

:thumbsup: / :thumbsdown: ?